homeassistant/components/homekit_controller/__init__.py

```diff
@@ -153,6 +177,23 @@ def get_characteristic_types(self):
         del hass.data[KNOWN_DEVICES][conn.unique_id]
         raise ConfigEntryNotReady
 
+    conn_info = conn.connection_info
+
+    device_registry = await dr.async_get_registry(hass)
+    device_registry.async_get_or_create(
+        config_entry_id=entry.entry_id,
+        connections={
+            (dr.CONNECTION_NETWORK_MAC, conn.unique_id),
```

Comment:
Is unique ID a Mac address?

Reply:
No - it looks like one and confuses everyone, but it isn't. I wanted to see whether the 'accessory id' made sense as a connection before i either moved it or added a connection type. I'm still not sure.

I think i lean towards it not have anything in connections, and then moving it to identifiers.

If i do move it to identifiers, could I do something like:

```python
        identifiers={
            (DOMAIN, 'serial-number', conn_info['serial-number']),
            (DOMAIN, 'accessory-id', conn.unique_id),
        },
```

Reply:
Yes, please do so. We should only have MAC addresses use the MAC type.

Reply:
Change made in 0263d5c.